inference-engine/include/ie_plugin_config.hpp

```diff
@@ -92,6 +92,7 @@ DECLARE_METRIC_KEY(FULL_DEVICE_NAME, std::string);
  *
  * The possible values:
  *  - "FP32" - device can support FP32 models
+ *  - "BF16" - device can support BF16 models
```

Comment:
may be `device can support BF16 computations for XX models`? XX can be replaced with FP32 or other actual type